src/interfaces/IIBCERC20.sol

```diff
@@ -5,12 +5,14 @@ import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 
 interface IIBCERC20 is IERC20 {
     /// @notice Mint new tokens to the ICS20Transfer contract
+    /// @param account Account to mint tokens to
     /// @param amount Amount of tokens to mint
-    function mint(uint256 amount) external;
+    function mint(address account, uint256 amount) external;
```

Comment:
I added this parameter because now the tokens are not escrow in he owner contract (`ICS20Transfer`), but in one of the escrow contracts.